ru/turing/courses/lesson3/Yemelyanov/Registry.java

```diff
@@ -0,0 +1,61 @@
+package ru.turing.courses.lesson3.Yemelyanov;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ *
+ * @author SteklotaraBlizz
+ *
+ */
+
+public class Registry<T extends String, T1 extends Alcohol> {
```

Comment:
```suggestion
public class Registry<String, T extends Alcohol> {
```
String - final класс. У него не может быть наследников. Ключ можно задать явно как String